test/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "declaration": false,
```

Comment:
Could you please include `"noEmit": true` so the tests don't get transpiled?

Reply:
Yes sorry